generics/services/survey.js

```diff
@@ -388,11 +388,70 @@ const userObservations = function(token, programId) {
     })
 }
 
+
+/**
+  * Get survey submissions by user.
+  * @function
+  * @name userSurveySubmissions
+  * @param {String} token - logged in user token.
+  * @returns {Promise} returns a promise.
+*/
+const userSurveySubmissions = function( token ) {
```

Comment:
@ankitpws Do we need this new API? Can we use the same as userSurveys?

Reply:
userSurveys Will return solutionId and programId and it will query survey collection but here we have get data from surveySubmission @aks30